Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm

```diff
@@ -263,7 +263,7 @@ static bool supportsClientSideAttachmentData(const Frame& frame)
         frame.editor().registerAttachmentIdentifier(attachment->ensureUniqueIdentifier(), contentType, defaultImageAttachmentName, buffer.copyRef());
         if (contentTypeIsSuitableForInlineImageRepresentation(contentType)) {
             auto image = HTMLImageElement::create(document);
-            image->setAttributeWithoutSynchronization(HTMLNames::srcAttr, DOMURL::createObjectURL(document, Blob::create(&document, buffer->extractData(), contentType)));
+            image->setAttributeWithoutSynchronization(HTMLNames::srcAttr, AtomString { DOMURL::createObjectURL(document, Blob::create(&document, buffer->extractData(), contentType)) });
```

Comment:
Makes me wonder if `createObjectURL` should just return an AtomString.

Reply:
It is unclear to me that we want URLs in the AtomStringTable in general. Here we need an AtomString because that's how store store Element attributes, but in general, I don't think we do.